djangosaml2/views.py

```diff
@@ -175,6 +177,9 @@ def login(request,
                     'target_url': result['url'],
                     'params': params,
                     })
+        except (DTDForbidden, EntitiesForbidden, ExternalReferenceForbidden):
+            return HttpResponseForbidden('<h1>403 Forbidden</h1>',
```

Comment:
Since we're not showing any specific page or message to the end user anyway, I suggest [raising `PermissionDenied`](https://docs.djangoproject.com/en/dev/ref/views/#the-403-http-forbidden-view) instead in order to support custom 403 error page.